packages/react-native-reanimated/android/build.gradle

```diff
@@ -468,7 +458,7 @@ task prepareReanimatedHeadersForPrefabs(type: Copy) {
 }
 
 tasks.preBuild {
-    dependsOn assertLatestReactNativeWithNewArchitecture, assertMinimalReactNativeVersion
+assertMinimalReactNativeVersion
```

Comment:
1. Let's add newlines
2. Does it work tho?
```suggestion
    dependsOn assertMinimalReactNativeVersion
```

Reply:
Thanks for pointing it out!